### PR TITLE
Admin feature mithun

### PR DESCRIPTION
**Pull request by mithun30052001**
Admin Feature added using Active_admin

### CODE REVIEW

File: app/admin/admin_users.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register AdminUser do
```
Comment (iMagesh) [inline app/admin/admin_users.rb:1]: please change the file name to users.rb

File: app/controllers/communities_controller.rb [hunk 2]
```diff
@@ -52,8 +54,24 @@ def set_community
     @community = Community.find(params[:id])
   end
 
+  def find_all_communities
+    @communities = Community.all
```
Comment (iMagesh) [inline app/controllers/communities_controller.rb:58]: Don't use .all method

File: app/controllers/public_controller.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 class PublicController < ApplicationController
 
   def index
-    @communities = Community.all.limit(5)
+    @communities = Community.all.order("post_count_this_week desc").limit(5)
```
Comment (iMagesh) [inline app/controllers/public_controller.rb:4]: .all is not required here

File: config/application.rb [hunk 4]
```diff
@@ -11,6 +11,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.time_zone = "Kolkata"
```
Comment (iMagesh) [inline config/application.rb:14]: should this be "Asia/Kolkata" ?

File: app/admin/admin_users.rb (hunk 1 above)
Comment (mithun30052001) [reply app/admin/admin_users.rb:1]: It is an auto-generated file used for different purpose

**Review by iMagesh — CHANGES_REQUESTED** [review]
Please check my comments and let me know once you have fixed it

File: config/application.rb (hunk 4 above)
Comment (vasanthakumar-a) [reply config/application.rb:14]: Resolved it

File: app/controllers/public_controller.rb (hunk 3 above)
Comment (vasanthakumar-a) [reply app/controllers/public_controller.rb:4]: Resolved it

File: app/controllers/communities_controller.rb (hunk 2 above)
Comment (vasanthakumar-a) [reply app/controllers/communities_controller.rb:58]: Communities will be displayed with an "infinite scroll feature" further

**Comment by mithun30052001** [comment]
Necessary changes made please check

**Pull request merged**